### PR TITLE
Apps: Sip

### PR DESCRIPTION
**Pull request by jamesjlyons**
## Description

Add these commands for the Sip color management app: show sip color picker, open contrast checker, check contrast between two hex values, and add hex color to sip history.

## Type of change

- [x] New script command

## Screenshot

CleanShot 2021-03-09 at 19 04 40@2x
CleanShot 2021-03-09 at 19 04 31@2x
CleanShot 2021-03-09 at 19 04 09@2x
CleanShot 2021-03-09 at 19 04 17@2x
CleanShot 2021-03-09 at 19 03 54@2x

## Dependencies / Requirements

Requires Sip v2.5.3 or above: https://sipapp.io/

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/sip/sip-add-hex-color.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Note: Sip X v2.5.3 required
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title  Add hex color to your history
```
Comment (thomaspaulmann) [inline commands/apps/sip/sip-add-hex-color.sh:7]:
```suggestion
# @raycast.title  Add Color to History
```

File: commands/apps/sip/sip-add-hex-color.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon ./images/sip.png
+# @raycast.packageName Sip
+# Documentation:
+# @raycast.author Sip
+# @raycast.authorURL https://twitter.com/sip_app/
+# @raycast.description  Add a color to your Sip history.
+# @raycast.argument1 { "type": "text", "placeholder": "Color Name" }
```
Comment (thomaspaulmann) [inline commands/apps/sip/sip-add-hex-color.sh:17]:
```suggestion
# @raycast.argument1 { "type": "text", "placeholder": "Name" }
```

File: commands/apps/sip/sip-check-contrast-hex.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Note: Sip X v2.5.3 required
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Check contrast between HEX color values
```
Comment (thomaspaulmann) [inline commands/apps/sip/sip-check-contrast-hex.sh:7]:
```suggestion
# @raycast.title Check Contrast
```

**Review by thomaspaulmann — APPROVED** [review]
@jamesjlyons these scripts are a wonderful addition to this repository and I'm sure many folks will love to use them. CC @ruiaureliano I think you enjoy those commands as well ;-)

**Comment by thomaspaulmann** [comment]
@jamesjlyons I tweaked the command names slightly but otherwise it looks good to go.

**Comment by Ansub** [comment]
is there is any way of getting Sip Extension through Store?

**Comment by jamesjlyons** [comment]
> is there is any way of getting Sip Extension through Store?

Definitely! I haven't made an extension yet but this would be a great one to start with. No guarantees on a timeline, but I should be able to start on it soon.

**Comment by Ansub** [comment]
ok thanks a lot James!

**Comment by thomaspaulmann** [comment]
@jamesjlyons check out the [Spotify Controls](https://www.raycast.com/thomas/spotify-controls) and [CleanShot](https://www.raycast.com/Aayush9029/cleanshotx) extensions. They are similar to your Sip scripts.

**Comment by jamesjlyons** [comment]
@Ansub The new Sip extension I made just got published on the store! [https://raycast.com/jameslyons/sip](https://raycast.com/jameslyons/sip)

**Pull request merged**